tests/a11y/selection-tests.js

```diff
@@ -53,12 +53,6 @@ test('static aria attributes are present', function (assert) {
   var selection = new BaseSelection($select, options);
   var $selection = selection.render();
 
-  assert.equal(
```

Comment:
How come this assertion needed to be removed? Should we add it back in somewhere else if it wasn't relevant?

Reply:
Good spot here @asmblah, it was removed because the SelectWoo team had updated this behaviour and moved the `combobox` role over to single select instances, I've updated the tests to put this assertion in the correct place 👍